internal/gitstrap/utils.go

```diff
@@ -20,3 +24,10 @@ func (g *Gitstrap) resolveOrg(m *spec.Model) string {
 	}
 	return m.Metadata.Owner
 }
+
+func (g *Gitstrap) getTeamOwner(m *spec.Model) (string, error) {
```

Comment:
@OrlovM I'd make it more generic, since we may need the same logic not only for "teams", e.g. `func requireOwner(meta *spec.Metadata) (string, error)`.

Also, the `*GItstrap` struct is not used here, so it can be not a method of this struct but just a function.

And the error could be more generic in case if we rename this method, e.g. `"Metadata owner is required"`.